dist/docs/guide/process/index.adoc

```diff
@@ -0,0 +1,49 @@
+## General provisioning process
+
+### Installation process
+
+Prospero delegates provisioning server to Galleon. When Galleon attempts to resolve components needed by the server, instead of using versions provided by the feature packs, Prospero intercepts that request and uses Wildfly Channels to find the correct version of the component and resolve it from the channels repositories.
+
+image::overview.png[]
```

Comment:
TODO - missing picture.

Reply:
sorry, it slipped the commit, pushed now